src/main/docs/guide/configuration.adoc

```diff
@@ -53,7 +53,7 @@ acme:
 <4> Enables ACME integration for micronaut. Default is `false`
 <5> Agrees to the Terms of Service of the ACME provider. Default is `false`
 <6> Location to store the certificate on the server.
-<7> Domain name for the certificate. Can with be a singular domain name (ex. domain.com) or a wildcard domain name (*.domain.com)
+<7> Domain name(s) for the certificate. Can with be multiple domain names (ex. test.domain.com,stage.domain.com) or a wildcard domain name (*.domain.com)
```

Comment:
I think folks could easily believe a comma separated list is supported because of the example. Perhaps in the configuration example use a list

Reply:
Fixed in 4ef616540cd0136007b3c1c548f535509f7290ae